zgw/verzoek/src/main/resources/dmn/zaakdossier_decision.dmn

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<definitions xmlns="https://www.omg.org/spec/DMN/20191111/MODEL/" xmlns:dmndi="https://www.omg.org/spec/DMN/20191111/DMNDI/" xmlns:dc="http://www.omg.org/spec/DMN/20180521/DC/" xmlns:modeler="http://camunda.org/schema/modeler/1.0" id="zaakdossiers_type_decision" name="Zaakdossiers Type decision" namespace="http://camunda.org/schema/1.0/dmn" exporter="Camunda Modeler" exporterVersion="5.2.0" modeler:executionPlatform="Camunda Cloud" modeler:executionPlatformVersion="8.0.0">
+  <decision id="zaakdossiers_type_decision_table" name="Zaakdossiers type decision_table">
```

Comment:
The DMN doesn't seem to be used inside the process yet

Reply:
<img width="912" alt="Screenshot 2023-01-23 at 08 34 13" src="https://user-images.githubusercontent.com/112691188/213995870-a8d905b1-7a2a-4022-b267-4049a51be738.png">

<img width="1066" alt="Screenshot 2023-01-23 at 08 35 22" src="https://user-images.githubusercontent.com/112691188/213996093-4c92c29d-4c52-4990-bcb5-ad6691ded0f0.png">


Not sure if this is the id that i should link